utils/firestore.js

```diff
@@ -996,62 +998,36 @@ const searchSpecimen = async (masterSpecimenId, siteCode) => {
 }
 
 const searchShipments = async (siteCode) => {
-    const snapshot = await db.collection('biospecimen').where('827220437', '==', siteCode).get();
+    const { reportMissingTube, submitShipmentFlag, no } = fieldMapping;
+    const healthCareProvider = fieldMapping.healthCareProvider.toString();
+    const tubesBagsCidKeys = swapObjKeysAndValues(tubesBagsCids);
+    const snapshot = await db.collection('biospecimen').where(healthCareProvider, '==', siteCode).get();
+
     if(snapshot.size !== 0){
-        //
-        return snapshot.docs.filter(document => {
-            
-            let data = document.data();
+        let shipmentData = [];
```

Comment:
This line can be moved before `if` block. The `else` block is not needed. Return `shipmentData` at end of function block.

Reply:
Makes sense to remove the else block after moving the `shipmentData` variable outside of the `if` block. I appreciate the suggestions.